app/views/pregnancies/_patient_dashboard.html.erb

```diff
@@ -1,3 +1,7 @@
+<div class="col-md-12">
+  <%= render partial: "pledge" %>
```

Comment:
@rebeccaestes Two things I'm wondering about here, would welcome your thoughts:

a) Does it make more sense to render this partial in `edit.html.erb` instead of burying it in a partial, since it's kind of its own separate thing? 

b) In the interest of making it consistent with other partials in this view, what do you think about passing a local into this partial? e.g. `render partial: 'pledge', locals: { pregnancy: @pregnancy }`, similar to how we do stuff in `edit`. Thoughts?